src/cozy/wrappers/import-page/import-page.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ImportPage from "../../react/components/ImportPage";
+import ReactWrapper from "../react-wrapper";
+
+// wrap original ImportPage component into
+const ImportPageWrapper = ({ client, bitwardenData, ...props }) => {
+  return (
+    <ReactWrapper client={client} bitwardenData={bitwardenData} {...props}>
+      <ImportPage></ImportPage>

```

Comment:
`<ImportPage />` :D

Reply:
Does that mean that if we have to import another react component, we'll hav to wrap it inside a ReactWrapper too? It means that we'll mount several `ReactWrapper` at the same time, right?

Reply:
Yep that's it. It is working for now and I didn't notice any performance issue. But we may have to find a better way in the future if we are limited by this architecture.